README.md

```diff
@@ -4,6 +4,7 @@
 
 [![crates.io](https://img.shields.io/crates/v/linfa.svg)](https://crates.io/crates/linfa)
 [![Documentation](https://docs.rs/linfa/badge.svg)](https://docs.rs/linfa)
+[![DocumentationLatest](https://img.shields.io/badge/docs-latest-blue)](https://rust-ml.github.com/linfa/rustdocs/linfa/)
```

Comment:
This gives me a 404 error

Reply:
yeah the rustdocs are built after the PR because we don't want to publish every branch/PR into `gh-pages` branch